dittybopper/syncer/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal
 
 RUN microdnf install git make tar
 WORKDIR /dittybopper-syncer
-RUN chmod 775 /dittybopper-syncer
-COPY entrypoint.sh /bin/entrypoint.sh
+COPY . /dittybopper-syncer/dashboards
+COPY dittybopper/syncer/entrypoint.sh /bin/entrypoint.sh
```

Comment:
This container build is not working for me...
```
STEP 1/8: FROM registry.access.redhat.com/ubi8/ubi-minimal
STEP 2/8: RUN microdnf install git make tar
--> Using cache d3d21a05b29fa1617f0b1c6ca0556bdd33db2d50f013f9d7b67524c102f0a342
--> d3d21a05b29
STEP 3/8: WORKDIR /dittybopper-syncer
--> Using cache bdee90ad2fa013d1f11dd728659eff56142ee95921363fa6e3ee394f63ffbccf
--> bdee90ad2fa
STEP 4/8: COPY . /dittybopper-syncer/dashboards
--> 1e9205394b7
STEP 5/8: COPY dittybopper/syncer/entrypoint.sh /bin/entrypoint.sh
Error: error building at STEP "COPY dittybopper/syncer/entrypoint.sh /bin/entrypoint.sh": checking on sources under "******/performance-dashboards/dittybopper/syncer": copier: stat: "/dittybopper/syncer/entrypoint.sh": no such file or directory

``

Reply:
I moved the build path to the root of the repository. Is that where you built it from?